apps/webstore/src/app/pages/auth/profile/profile.component.html

```diff
@@ -171,7 +171,7 @@ <h2 i18n="@@account-info">Account info</h2>
                 i18n="@@invalid-street-address"
                 data-cy="account-invalid-address-2"
             >
-                The address is too long.
+                Invalid address
```

Comment:
The alert should say why it is invalid
```suggestion
                The address is too long
```